themes/docs-new/layouts/shortcodes/cookbooks_summary.md

```diff
@@ -1,13 +1,10 @@
-A cookbook is the fundamental unit of configuration and policy distribution.
+A cookbook is the fundamental unit of configuration and policy distribution in Chef Infra.
 
 A cookbook defines a scenario and contains everything that is required to support that scenario:
 
-- Recipes that specify the resources to use and the order in which
-    they are to be applied
-- Attribute values
-- File distributions
-- Libraries
-- Custom Resources
-- Templates
-- Ohai Plugins
-- The `metadata.rb` file
\ No newline at end of file
+- Recipes that specify Chef Infra built-in resources to use and the order in which they are to be applied
+- Attribute values allowing different configuratinos to be applied based on environments such as dev vs. production.
+- Custom Resources used to extend Chef Infra beyond the built-in resources.
+- Files and Templates for distribution to systems.
+- Custom Ohai Plugins for extending system configuration collection.
+- The `metadata.rb` file describing the cookbook itself and any dependendencies it may have.
```

Comment:
```suggestion
- The `metadata.rb` file, which describes the cookbook itself and any dependencies it may have.
```

Reply:
Do you not like my creative way of spelling that?